src/encoded/static/components/typeutils.js

```diff
@@ -295,44 +295,39 @@ export function fileStatusList(session, sessionProperties) {
 }
 
 
-// Display supersedes/superseded_by lists.
-export const Supersede = ({ context }) => {
+/**
+ *  Display supersedes/superseded_by/alternate_accessions lists.
+ */
+export const ReplacementAccessions = ({ context }) => {
```

Comment:
Not a big deal
- Need JSDoc?

Reply:
I haven’t been putting that on React component comments because we already document props in propTypes. Not sure if JSDoc has been updated for React components.